src/test/java/com/github/shyiko/mysql/binlog/BinaryLogFileReaderIntegrationTest.java

```diff
@@ -42,12 +42,32 @@
     public void testNextEvent() throws Exception {
         BinaryLogFileReader reader = new BinaryLogFileReader(new GZIPInputStream(
                 new FileInputStream("src/test/resources/mysql-bin.sakila.gz")));
+        readAll(reader, 1462);
+    }
+
+    @Test
+    public void testNoChecksum() throws Exception {
+        EventDeserializer eventDeserializer = new EventDeserializer();
+        BinaryLogFileReader reader = new BinaryLogFileReader(new GZIPInputStream(
+                new FileInputStream("src/test/resources/mysql-bin.sakila.gz")), eventDeserializer);
+        readAll(reader, 1462);
+    }
+
+    @Test
+    public void testCRCChecksum() throws Exception {
+        EventDeserializer eventDeserializer = new EventDeserializer();
+        BinaryLogFileReader reader = new BinaryLogFileReader(
+                new FileInputStream("/data/mysql-bin.000001"), eventDeserializer);
```

Comment:
test data must reside in src/test/resources/

Reply:
done